chunks/lang-rust/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM ${base}
 USER gitpod
 
 # Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
-ENV TRIGGER_REBUILD=4
```

Comment:
I think https://github.com/gitpod-io/workspace-images/pull/1013 would be more appropriate.

Reply:
:wave: how so? I'm not sure I follow.

Reply:
It's comes first before this PR. Maybe add `co-authored-by` in commit.

Reply:
:heart_eyes: good call, thank you, @gengjiawen for sharing. I will rebase the commit, and include [c3potheds](https://github.com/c3potheds), too. FYI, it'll get squashed with other commits.